src/main/java/teammates/ui/controller/InstructorFeedbackRemindParticularStudentsPageData.java

```diff
@@ -1,14 +1,43 @@
 package teammates.ui.controller;
 
+import java.util.List;
+import java.util.ArrayList;
 import teammates.common.datatransfer.AccountAttributes;
 import teammates.common.datatransfer.FeedbackSessionResponseStatus;
+import teammates.ui.template.RemindParticularStudentsCheckboxEmailNamePair;
 
 public class InstructorFeedbackRemindParticularStudentsPageData extends PageData {
     public FeedbackSessionResponseStatus responseStatus;
     public String courseId;
     public String fsName;
+    private List<RemindParticularStudentsCheckboxEmailNamePair> emailNamePairs;
```

Comment:
I tried to keep them private but it cascades into many files and generates about 18 errors. I'll try to do it again tomorrow morning and see if there is reasonable way to remove the errors.